code/modules/antagonists/eldritch_cult/eldritch_effects.dm

```diff
@@ -7,7 +7,6 @@
 	layer = SIGIL_LAYER
 	///Used mainly for summoning ritual to prevent spamming the rune to create millions of monsters.
 	var/is_in_use = FALSE
-
 /obj/effect/eldritch/Initialize()
```

Comment:
No newline uwu?

Reply:
uwah

Reply:
No need to save on lines. It's better if there's some sort of separation, so it's clear where one definition ends and when another starts.